service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadAdditionalApplicationsSubmittedControllerTest.java

```diff
@@ -223,6 +219,7 @@ void submittedEventShouldNotifyHmctsAdminAndRepresentativesWhenCtscToggleIsDisab
                     .document(ORDER)
                     .supplementsBundle(new ArrayList<>())
                     .others(List.of(element(other)))
+                    .respondents(List.of(respondents.get(0), respondents.get(1), respondents.get(3)))
```

Comment:
If the constants are `Element<Respondent>` instead of `Respondent`, then you can just use the constants here (instead of having to use the index in the list. I think it would improve readability.